### DISCUSSION TITLE
Webhooks + Service Account Settings Disappear from the settings page in The Saleor Dashboard

### DISCUSSION THREAD

**Discussion by Adeolamorren** [saleor/saleor#6169] [Q&A]
Hi Everybody,

Webhooks + Service Account Settings seem to be disappearing from the settings page in The Saleor Dashboard  after I deploy to heroku using the Saleor Guide.

### Steps to reproduce the problem
1. Install Saleor Locally using the official Saleor Guide
2.  Deploy to Heroku using the Saleor Developer Guide as well

### Screenshots

![Screen Shot 2020-09-25 at 12 57 18 PM](https://user-images.githubusercontent.com/54296502/94325201-22889900-ff52-11ea-8bd1-f94e814ea7c3.png)

**Comment by patrys** [comment]
The version of the Dashboard you're running has a dedicated Apps section in its sidebar.
